go/vt/vtgate/plan_execute.go

```diff
@@ -92,6 +91,7 @@ func (e *Executor) newExecute(
 		plan               *engine.Plan
 	)
 
+	var cancel context.CancelFunc
```

Comment:
really nitpicking here, but this should move up into the rest of the variables declared outside the for loop